### PR TITLE
Add Roon media_player integration

### PR DESCRIPTION
**Pull request by pavoni**
## Proposed change

Add documentation for the new Roon Labs media player integration,

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/37553

- Link to parent pull request in the Brands repository: 
https://github.com/home-assistant/brands/pull/1769

- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/roon.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+</div>
+
+### Configuration
+
+<div class='note'>
+
+  You need the Hostname or IP adresss of the machine that runs your Roon Core. This might be a machine name (which can be followed buy .local, eg myserver.local) or can be an IP address.
+
+</div>
```
Comment (frenck) [inline source/_integrations/roon.markdown]: Do we really need 2 note blocks? I think this can just be part of the normal text.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by pavoni** [comment]
@frenck Just wanted to check I've updated this correctly (the integration is ready to merge - so hoping it will make it to 0.115)

File: source/_integrations/roon.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+If you use an IP address please assign a static IP address to the machine that runs Roon Core. This ensures that it won't change IP addresses, so you won't have to change the configuration in Home Assistant if it reboots and changes IP address. See your router's manual for details on how to set this up.
+
+</div>
+
+### Configuration
+
+You need the Hostname or IP adresss of the machine that runs your Roon Core. This might be a machine name (which can be followed buy .local, eg myserver.local) or can be an IP address.
+
+1. From the Home Assistant front-end, navigate to 'Configuration' then 'Integrations'. Under 'Set up a new integration' locate 'Roon' and click 'Configure'.
```
Comment (klaasnicolaas) [inline source/_integrations/roon.markdown]:
```suggestion
1. From the Home Assistant front-end, navigate to **Configuration** then **Integrations**. Under **Set up a new integration** locate 'Roon' and click **Configure**.
```

File: source/_integrations/roon.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: RoonLabs Media Player
+description: Instructions on how to integrate a RoonLabs multi room player into Home Assistant.
+ha_category:
+  - Media Player
+ha_release: 0.114
```
Comment (klaasnicolaas) [inline source/_integrations/roon.markdown]:
```suggestion
ha_release: 0.115
```

File: source/_integrations/roon.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+If you use an IP address please assign a static IP address to the machine that runs Roon Core. This ensures that it won't change IP addresses, so you won't have to change the configuration in Home Assistant if it reboots and changes IP address. See your router's manual for details on how to set this up.
+
+</div>
+
+### Configuration
+
+You need the Hostname or IP adresss of the machine that runs your Roon Core. This might be a machine name (which can be followed buy .local, eg myserver.local) or can be an IP address.
+
+1. From the Home Assistant front-end, navigate to 'Configuration' then 'Integrations'. Under 'Set up a new integration' locate 'Roon' and click 'Configure'.
+2. Enter the Hostname or IP address for the Roon Core machine and click 'Submit'.
```
Comment (klaasnicolaas) [inline source/_integrations/roon.markdown]:
```suggestion
2. Enter the `Hostname` or `IP address` for the Roon Core machine and click **Submit**.
```

File: source/_integrations/roon.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+3. Home Assistant will then contact your Roon Core and ask to be authorized. You will need to enable this extension in the Room Application. Go to 'Settings' and then 'Extensions', there you will see an entry for Home Assistant with a button next to it. Click 'Enable'.
```
Comment (klaasnicolaas) [inline source/_integrations/roon.markdown]:
```suggestion
3. Home Assistant will then contact your Roon Core and ask to be authorized. You will need to enable this extension in the Room Application. Go to **Settings** and then **Extensions**, there you will see an entry for Home Assistant with a button next to it. Click **Enable**.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by pavoni** [comment]
Thanks @klaasnicolaas

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍
⚠️ The ha_release might need an update before merging this one in. (maybe not 😄 )

**Review by frenck — APPROVED** [review]

**Pull request merged**